lib/tasks/index.rake

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'arclight'
+
+# hardcoded since we don't have access to Blacklight.connection_config[:url] here
+SOLR_URL = ENV['SOLR_URL'] || 'http://127.0.0.1:8983/solr/blacklight-core'
```

Comment:
And maybe that line just lives in the `load_indexer` method below?  Just thinking out-loud.